features/step_definitions/developer_portal/cms_toolbar_steps.rb

```diff
@@ -1,9 +1,14 @@
 # frozen_string_literal: true
 
-Given "they visit the developer portal in CMS mode" do
+Given /^they visit the developer portal in CMS mode(\swith an expired token)?/ do |token_is_expired|
+  cms_token = @provider.settings.cms_token!
+  expires_at = Time.now.utc
+  expires_at += 1.minute.to_i unless token_is_expired
```

Comment:
I'm worried about transient errors where we open portal at the same second the token was generated. Perhaps remove from `Time.now` to be sure?

Reply:
Not sure if I understood what you meant, but I did this: https://github.com/3scale/porta/pull/3932/commits/c633f64dfe5b3723551890e4433fa58db4fca03c